admin/class-admin-editor-specific-replace-vars.php

```diff
@@ -0,0 +1,257 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Determines the editor specific replacement variables.
+ */
+class WPSEO_Admin_Editor_Specific_Replace_Vars {
+
+	/**
+	 * @var array The editor specific replacement variables.
+	 */
+	protected $editor_specific_replace_vars = array(
+		// Posts types.
+		'page'                    => array( 'id', 'pt_single', 'pt_plural', 'parent_title' ),
+		'post'                    => array( 'id', 'term404', 'pt_single', 'pt_plural' ),
+		// Custom post type.
+		'custom_post_type'        => array( 'id', 'term404', 'pt_single', 'pt_plural' ),
```

Comment:
A custom post type could be hierarchical and thus have a parent_title, should this be taken into consideration?

Reply:
You're right, I forgot that one in my code review. Sorry @igorschoester

Reply:
Ah, will add. Could do a check with `is_post_type_hierarchical` too. Seems a little overboard for now though.